msal-auth-node/app.js

```diff
@@ -0,0 +1,41 @@
+/*
+ This application demonstrates how to issue a call to a protected API.
```

Comment:
The old text was a bit better, in that this file isn't showing how to issue a call to a protected API.  It's about protecting an API route.

Reply:
ok reverted